## Background:
[Control-FREEC](http://boevalab.inf.ethz.ch/FREEC/index.html) is a tool for assessing copy number and allelic content 
using next generation sequencing data. The tool provides the script [makeGraph.R](https://github.com/BoevaLab/FREEC/blob/master/scripts/makeGraph.R) 
to visualize normalized copy number profile with predicted Copy Number Alterations (CNAs). However, this script generates different plots for different chromosomes and the Y-scale could be different for each chromosome based on the distribution of the data.

An example plot for the [Control-FREEC test data](http://boevalab.inf.ethz.ch/FREEC/tutorial.html#Example) as generated by **makeGraph.R** is shown below:

```
# control-freec command line:

freec -conf  config_ctrl.txt
cat makeGraph.R | R --slave --args 2 HCC1143.arachne_ratio.txt
```

![**Figure A**](/data/HCC1143.arachne_ratio.txt.png) 

## Motivation:
For the genome-wide CNA visualization, we needed a scaled plot with combined chromosomes and uniform Y-scale. This script is inspired by the original [makeGraph.R](https://github.com/BoevaLab/FREEC/blob/master/scripts/makeGraph.R)
and attempts to achieve combined chromosome plot with uniform Y-scale.

## How to run:
I have developed alternative R-script [plot_controlfreec.R](https://github.com/sagarutturkar/R-scripts/blob/master/plot_controlfreec.R) that uses famous ggplot2 to plot the same data in a different style.  
This script generates the plot for log2 scale as well.  
Plotting of the BAF is currently not supported. Stay tuned ...

```
Rscript --vanilla  plot_controlfreec.R -i HCC1143.arachne_ratio.txt -p ./ -n HCC1143

# Parameter description:
#-i = input file *_ratio.txt as generated by control-freec run
#-p = output path
#-n = NAME
```

![**Figure B**](/data/HCC1143_plot.png) 

